conditions/README.md

```diff
@@ -0,0 +1,341 @@
+    </effectiveTime>
+    <availabilityTime value="20100113140445"/>
+    <pertinentInformation typeCode="PERT">
+        <sequenceNumber value="+1"/>
+        <pertinentAnnotation classCode="OBS" moodCode="EVN">
+            <text>Problem Info: Related Problem: Child of Allergy to aspirin Problem Notes: This is a peanut allergy
+                (Evolved into H/O: aspirin allergy 13-Jan-2010)
+            </text>
+        </pertinentAnnotation>
+    </pertinentInformation>
+</ObservationStatement>
+</component>
+```
+
+</details>
+
+4. Where `extension[index].url` equals `"https://fhir.hl7.org.uk/STU3/StructureDefinition/Extension-CareConnect-ProblemSignificance-1"`
+5. For each `extension[index].value`, where `extension[index].url` equals `"https://fhir.hl7.org.uk/STU3/StructureDefinition/Extension-CareConnect-RelatedClinicalContent-1"`
+   or `"https://fhir.hl7.org.uk/STU3/StructureDefinition/Extension-CareConnect-ActualProblem-1"`  
+   and:`extension[index].value` does not reference:
+- a [List](../list/README.md)
+- an [Encounter](../encounters/README.md)
+  or:
+- a suppressed [MedicationRequest](../medication%20requests/README.md) (with `MedicationRequest.status` == `"stopped"` and `MedicationRequest.intent` == `"order"`)
+
+6. where `extension[index].url` equals `"https://fhir.hl7.org.uk/STU3/StructureDefinition/Extension-CareConnect-ActualProblem-1"`
+7. where `extension[index].url` equals `"https://fhir.hl7.org.uk/STU3/StructureDefinition/Extension-CareConnect-RelatedProblemHeader-1"`
+
+#### CD for transformed actual problem header
+
+| Mapped to (XML HL7 CD) | Mapped from (JSON FHIR / other source )         |
+|------------------------|-------------------------------------------------|
+| code \[@code]          | fixed value = `"55607006"`                      |
+| code \[@codeSystem]    | fixed value = `"2.16.840.1.113883.2.1.3.2.4.15` |
+| code \[@displayName]   | fixed value = `"Problem"`                       |
+
+#### Pertinent Annotation Text for Generated Observation 
+
+Where the condition has an Actual Problem extension <sup>6</sup> and the referenced problem is a `AllergyIntolerance`, 
+`Immunization` or `MedicationRequest` the following will be added to `pertinentInformation / pertinentAnnotation / text`:
+
+- `"Transformed {{resourceType}} problem header"` where `{{resourceType}}` is the resource type of the actual problem
+- `"Originally coded: {{codeText}}"` where `{{codeText}}` is `Condition.code.text` or 
+`Condition.coding[0].extension[index].extenstion[index2].value` where `extension[index].url` == `https://fhir.nhs.uk/STU3/StructureDefinition/Extension-coding-sctdescid`
+and `extension[index2].url` == descriptionDisplay. If the text field or the description display extension are not populated
+the string will not be added.
+
+Where `Condition.note` contains annotations, `annotation.text` will be added in the following manner:
+
+- `"Problem Notes: {{annotation1}}, {{annotation2}}, {{annotation3}} ... "` 
```

Comment:
Minor: It appears to be using a semi-colon instead of a comma.

Reply:
I'll change it, thanks